pkg/operator/starter.go

```diff
@@ -246,6 +244,8 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 	).WithConditionalResources(
 		bindata.Asset,
 		[]string{
+			"assets/openshift-controller-manager/old-leader-role.yaml",
```

Comment:
Let's also add a TODO when these should be removed

I guess we can also remove the `leader-ingress-to-route-controller` files later, once this PR merges.

Reply:
This change will be present in 4.20, so we can remove the files in 4.21.

Reply:
Ammeded the last commit.